Source/ACE/Entity/Player.cs

```diff
@@ -45,6 +45,11 @@ public sealed class Player : MutableWorldObject
         public bool IsOnline { get; private set; }
 
         /// <summary>
+        /// ObjectId of the currently selected Target (only players and creatures)
+        /// </summary>
+        public uint SelectedTarget { get; set; }

```

Comment:
I needed the server to track it, so I could issue the smite command to the selected target. In retail there are even messages which lead to store the target as last attacker in the properties of a character